modules/component/src/componentNext.ts

```diff
@@ -98,13 +86,38 @@ export class ComponentNext<P1 extends ComponentProps> {
     )
   }
 
+  /**
+   * Creates a new component with initial state as undefined
+   *
+   * ```typescript
+   * import {ComponentNext} from '@action-land/component'
+   *
+   * const component = ComponentNext.empty // creates new component with initial state as undefined
+   * ```
+   */
   static get empty(): ComponentNext<{
     iState: undefined
     oView: void
   }> {
     return ComponentNext.lift(undefined)
   }
 
+  /**
+   * Adds ability to transform component's state matching action type
```

Comment:
Couple of things:
- imperative voice,
- action-type is a confusing idea since it's typescript. Here we mean the type _value_ of the action, for example but people might confuse it with the Type of the Action.

Reply:
Recommended sentence would be something like:

- Add transformation of component's state for a given action.